pkg/bpmn/flow_node_executor.go

```diff
@@ -0,0 +1,86 @@
+package bpmn
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pbinitiative/zenbpm/pkg/bpmn/model/bpmn20"
+	"github.com/pbinitiative/zenbpm/pkg/bpmn/runtime"
+	"github.com/pbinitiative/zenbpm/pkg/storage"
+)
+
+// FlowNodeExecutor defines the behavior of all BPMN executors
```

Comment:
```suggestion
// FlowNodeInstanceExecutor defines the behavior of all BPMN executors
```
use correct function name